integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/jms/securevault/password-tmp

```diff
@@ -0,0 +1 @@
+wso2carbon
```

Comment:
What is the purpose of this file?

Reply:
This is the keystore/primary key password, required when using Cipher Tool/Secure Vault.

Reply:
:+1:

Reply:
you will have to update the licence